### PR TITLE
Fix StyleCop errors

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Summary
This PR addresses the StyleCop warnings that appear every time you run build.cake.

- Line endings fixed
- Multiple blank lines removed
- Using statement moved outside namespace declaration
- Renamed jToken to jwtToken so it isn't flagged as hungarian notation.
- csproj file now uses PackageLicenseFile and LicenseFile elements

Only the "CSC : warning SA0001: XML comment analysis is disabled due to project configuration" warning is left.

Fixes N/A

## Type of PR
- [ ] Bug Fix (non-breaking fixes to existing functionality)
- [ ] New Feature (non-breaking changes that add new functionality)
- [ ] Documentation update
- [ ] Test Updates
- [x] Cosmetic 

## Test Information
- [ ] My PR required test updates 

.NET Version:
Os Version:

## Signoff
- [ ] I have submitted a CLA for this PR
- [ ] Each commit message explains what the commit does
- [ ] I have updated documentation to explain what my PR does
- [ ] My code is covered by tests if required
- [x] I checked StyleCop warnings on my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,26 +63,13 @@ Once you initialize a `Client`, you can call methods to make requests to the Okt
 
 ```csharp
 var client = new IdxClient(new IdxConfiguration()
-            {
-                Issuer = "{YOUR_ISSUER}", // e.g. https://foo.okta.com/oauth2/default, https://foo.okta.com/oauth2/ausar5vgt5TSDsfcJ0h7
-                ClientId = "{YOUR_CLIENT_ID}",
-                ClientSecret = "{YOUR_CLIENT_SECRET}", //Required for confidential clients. 
-                DeviceToken = "{YOUR_DEVICE_TOKEN}", // Client device token. Optional property. Only used for confidential clients with ClientSecret.
```
Comment (laura-rodriguez) [inline README.md:70]: Did you mean to remove these sections?

File: samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config [hunk 2]
```diff
@@ -1,131 +1,131 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
-  For more information on how to configure your ASP.NET application, please visit
-  https://go.microsoft.com/fwlink/?LinkId=301880
-  -->
-<configuration>
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config:6]: If there are no important changes in the web.config, can we please exclude it from the PR?

File: README.md (hunk 1 above)
Comment (danielmaharry-okta) [reply README.md:70]: No I didn't mean to remove the Custom Headers section or the Device Token line. 
Not sure how it happened. Have it added it back in.

File: samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config (hunk 2 above)
Comment (danielmaharry-okta) [reply samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config:6]: @laura-rodriguez The change to v3.6 is important. It's the only way I have found to compile the samples without errors. If I leave it at 2.0.1 and compile, VS does not generate the required Roslyn directories while building the sample.

File: samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget/Web.config [hunk 3]
```diff
@@ -112,8 +112,8 @@
...
-      <compiler language="vb;vbs;visualbasic;vbscript" extension=".vb" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.VBCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=2.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" warningLevel="4" compilerOptions="/langversion:default /nowarn:41008 /define:_MYTYPE=\&quot;Web\&quot; /optionInfer+" />
+      <compiler extension=".cs" language="c#;cs;csharp" warningLevel="4" compilerOptions="/langversion:7.3 /nowarn:1659;1699;1701;612;618" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.CSharpCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=3.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+      <compiler extension=".vb" language="vb;vbs;visualbasic;vbscript" warningLevel="4" compilerOptions="/langversion:default /nowarn:41008,40000,40008 /define:_MYTYPE=\&quot;Web\&quot; /optionInfer+" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.VBCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=3.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
```
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget/Web.config:116]: Sure - VS added it in automatically when I upgraded the package from 2.0.1 to 3.6.
WIll remove it

File: samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config (hunk 2 above)
Comment (danielmaharry-okta) [reply samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config:6]: please ignore. Have removed vb line from bottom of web.config line as requested elsewhere.

**Pull request closed without merging**